packaging/linux/isledecomp.desktop.in

```diff
@@ -0,0 +1,35 @@
+[Desktop Entry]
+Version=1.5
+
+Name=@APP_NAME@
+Comment=@APP_SUMMARY@
+
+Icon=@APP_ID@
+Type=Application
+Categories=Game;KidsGame;AdventureGame;Qt
+
+Keywords=LEGO;lego;LEGO Island
+Keywords[da]=LEGO;lego;Panik på LEGO Øen
+Keywords[de]=LEGO;lego;Abenteuer auf der LEGO Insel
+Keywords[es]=LEGO;lego;La Isla LEGO
+Keywords[fr]=LEGO;lego;Aventures sur L'île LEGO
+Keywords[it]=LEGO;lego;Isola LEGO
+Keywords[ja]=LEGO;lego;レゴアイランドの大冒険
+Keywords[ko]=LEGO;lego;레고 아일랜드
+Keywords[pt]=LEGO;lego;A Ilha LEGO
+Keywords[ru]=LEGO;lego;Остров LEGO
+
+SingleMainWindow=true
+
+TryExec=isle
+Exec=isle
+
+Actions=play;configure
+
+[Desktop Action play]
+Name=Play Game
+Exec=isle
+
+[Desktop Action configure]
+Name=Configure Settings
+Exec=config
```

Comment:
```suggestion
Exec=isle-config
```

Reply:
The "Exec" key is actually the command used to launch an action. It can be pulled from the $PATH or you can use an absolute path. In the Flatpak, for example, this expands to `/app/bin/config`.

If we really want this name, then I can just update the target name or the install directive in CMake and all builds will use it. Honestly, I think this is the path we should take since `config` is way too general of a name anyway. While Flatpak builds won't suffer from name collision issues, non-sandboxed packages absolutely will.

Reply:
We renamed `config` to `isle-config` recently.
https://github.com/isledecomp/isle-portable/commit/73bab247218fee0f6e5e555a08c6a88a33f105e5

Reply:
Ah, gotcha. I haven't merged `master` in a few days. I'll get that synced up.